packages/component-library/src/props/modalProps.ts

```diff
@@ -0,0 +1,11 @@
+import { type CSSProperties } from 'react';
+
+export type BasicModalProps = {
```

Comment:
Can we move this to `src/Modal.ts` instead? For now it might only have a type, but eventually I think we could (and should) add the component there too.